crypto/dso/dso_win32.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include "dso_locl.h"
+#include "e_os.h"
```

Comment:
But on less side-note-ish note, it wouldn't be inappropriate to aim for order *preservation* in this request. And since original cryptlib.h was including e_os.h *prior* all other openssl/\*.h it would be more appropriate to keep this pattern. I mean in multiple cases in this request we observe
```
#include "internal/cryptlib.h"
#include "e_os.h"
```
while less intrusive order would be
```
#include "e_os.h"
#include "internal/cryptlib.h"
```
At least we could be more confident that there are no new subtle issues...